tests/acceptance/link-test.js

```diff
@@ -1,7 +1,7 @@
-import { describe, it, beforeEach, afterEach } from 'mocha'
-import { expect } from 'chai'
-import startApp from '../helpers/start-app'
 import destroyApp from '../helpers/destroy-app'
+import startApp from '../helpers/start-app'
+import { expect } from 'chai'
```

Comment:
I would vote no. Our current eslint rule only enforces consistency, i.e. `import {expect } from 'chai'` will fail, but I've been removing the extra padding spaces when I see them (they were generated that way but in our new generators I'm not adding the extra spaces).